libs/internal/src/serialization/json_segment.cpp

```diff
@@ -20,7 +20,10 @@ tl::expected<std::optional<data_model::Segment::Target>, JsonError> tag_invoke(
 
     data_model::Segment::Target target{};
 
-    PARSE_FIELD_DEFAULT(target.contextKind, obj, "contextKind", "user");
+    // The zero value of a ContextKind ("" - empty string) is not valid in the
+    // domain of possible contexts. This field is parsed as REQUIRED to
+    // specify that fact.
+    PARSE_REQUIRED_FIELD(target.contextKind, obj, "contextKind");
```

Comment:
This can be required since Targets will always have a contextKind. Users are added in the `included` or `excluded` fields (and omitted from `includedContexts` and `excludedContexts`).